src/webapp/js/actions/index.js

```diff
@@ -85,6 +86,22 @@ export function signoutUser() {
 }
 
 export function resetLogoutCounter() {
+  const refreshToken = localStorage.getItem('refresh_token');
```

Comment:
all of the methods above are using redux thunk, change back dispatcher to dispatch and use it here too